### DISCUSSION TITLE
Wrapping cursor when translating/rotating/scaling in editor (4.0)

### DISCUSSION THREAD

**Discussion by stephen-berry** [godotengine/godot-proposals#4086] [Editor]
So just as the title says, Godot has recently allowed us to add keys to act like blender where you can hit a key to translate, rotate, or scale your items in the editor. Unlike blender though, Godot doesn't have the cursor wrap back around the screen once you hit the end. This means that we have to accept changes, and then move mouse back to screen, and repeat the process. 

Wasn't sure if this was something that'd be considered a feature, improvement, or what so....thoughts? Didn't find any similar issues posted when I looked, so sorry if this has been discussed elsewhere already :)

**Comment by eon-s** [comment]
The mouse wrap function exists and is used for panning, I guess that it could be easily added the other actions.

I guess that for something like this, translate may have to follow the node at the point of wrapping (sounds tricky to make it right), and scale can zoom in/out?

**Comment by Calinou** [comment]
cc @rcorre, as they originally implemented the feature in https://github.com/godotengine/godot/pull/56543.

**Comment by rcorre** [comment]
Good idea! It might be best to approach after https://github.com/godotengine/godot/pull/58389 as that refactors some of the code.
